shared/checks/oval/installed_app_is_rhv4.xml

```diff
@@ -0,0 +1,30 @@
+<def-group>
+  <definition class="inventory"
+  id="installed_app_is_rhv4" version="1">
+    <metadata>
+      <title>Red Hat Virtualization 4</title>
+      <affected family="unix">
+        <platform>multi_platform_all</platform>
+      </affected>
+      <reference ref_id="cpe:/a:redhat:virtualization:4"
+      source="CPE" />
+      <description>The application installed installed on the system is
+      OpenShift 3.</description>
```

Comment:
openshift v rhvm

Reply:
I don't think that it is that obvious. If you install on `rhel-h`, is rhv now an application?